src/backend/base/langflow/custom/custom_component/custom_component.py

```diff
@@ -59,6 +59,8 @@ class CustomComponent(BaseComponent):
     is_input: bool | None = None
     """The input state of the component. Defaults to None.
     If True, the component must have a field named 'input_value'."""
+    add_tool_output: bool | None = False
+    """The tool state of the component. Defaults to False."""
```

Comment:
By `tool state`, you mean "whether the component will be treated as a tool"?

Reply:
yes

Reply:
Maybe we should change the documentation to that. Being more verbose is fine

Reply:
Added!